ibm/service/vpc/resource_ibm_is_instance_template.go

```diff
@@ -437,6 +448,14 @@ func ResourceIBMISInstanceTemplate() *schema.Resource {
 							Optional: true,
 							Computed: true,
 						},
+						isInstanceTemplateBootVolumeTags: {
+							Type:        schema.TypeSet,
```

Comment:
Do we have update support here?

Reply:
https://pages.github.ibm.com/vpc/vpc-spec-artifacts/branch/master/public/2022-03-29.html#/Instances/update_instance_template
We dont have update support here..